server/lib/nl/common/utils.py

```diff
@@ -31,6 +32,9 @@
 
 # TODO: Consider tweaking/reducing this
 NUM_CHILD_PLACES_FOR_EXISTENCE = 15
+# List of date preps that indicate single date
```

Comment:
what is preps?

Reply:
it's a field in the date object: https://github.com/datacommonsorg/website/blob/master/server/lib/nl/detection/types.py#L322

and I believe it's short for preposition